03-leadership-and-inclusion/code-journals.md

```diff
@@ -0,0 +1,43 @@
+# Code Journals
+
+## Purpose
+
+The purpose of the Code Journal is to give each individual at least one tool for self-reflection. With the practice of consistently filling out the code journal, we hope to accomplish the following specific things:
+
+- Teach and practice self-reflection
+- Reinforce weekly learning goals
+- Give you an artifact to catalog and review your learnings over time
+
+**An Ada instructor will never ask to see your code journal.**
+
+## How
+
+The code journal is yours. Feel free to fill it out and use it however is most useful for you. Use markers and draw and bullet journal if that's what works best for you! Keep it clean and orderly if that's what you'd like, or scribble in it and draw pictures if you need to.
+
+At Ada, we'll ask that each person is intentional about how they reflect, though. It may be useful to use the following template for your weekly entry. This template emphasizes reflecting on the learning goals.
+
+---
+
+**Week #X**
+
+Question | My Response |
+--- |
+fill in Learning Goal #0 | fill in your reflection on Learning Goal #0
+... | ...
+fill in Learning Goal #(X-1) | fill in your reflection on Learning Goal #(X-1)
+What went well for me this week? |
+What concepts was the most difficult for me this week? |
+What are things I'd like to keep doing? |
+What are things I'd like to change or start doing? What are things I need to do better on? |
+
+---
+
+### Where can I find the Learning Goals for the week?
+
+You can find learning goals listed at the beginning of each lecture.
```

Comment:
Also might be good to add a note about the weekly learning goals in the pedagogy repo